### ISSUE TITLE
Feature Request: Quick Edit: better flow !

### ISSUE THREAD

**Issue by bonnebulle** [wekan/wekan#4990]
Am using last version (Docker install, 64bit):
```
WeKan ® Version | 6.98.0
Version de Meteor | 2.12
Version de Node | 14.21.3
Version de MongoDB | 6.0.7
```

# actual feature
Click on card open Details panel (left)
Click make drag-n-drop possible

# issue
I cant see my board and need to do 2 more clicks to edit the title (text)
So, I lost my flow (continuity of information)

# Feature Request
Have an option to do :
- one or double click = change the title (text) directly from board !
- add a drag-n-drop icon (or mod)

# what I already do
- I search all options/config/parameters,Can't find it !
- I search for issues here ( find this, neerly [ask the same](https://github.com/wekan/wekan/discussions/4244) )

# inspiration
https://nullboard.io/preview

**Comment by xet7** [comment]
Hmm. Do you mean this feature?

https://github.com/wekan/wekan/wiki/Drag-Drop

**Comment by xet7** [comment]
Maybe related https://github.com/wekan/wekan/discussions/4244

Please add some screenshot mockups. I try understand, while trying to wake up and drink coffee.

**Comment by bonnebulle** [comment]
Yep, like I mentioned this ( https://github.com/wekan/wekan/discussions/4244 ) 
it's nearly the same (not exactly)
...
For Resume :
I just want to one click on a card and, *bim*, it's directly allow me to edit the text/title !
Directly from the board... not from the detailed left panel.
... 
A quick way to do modifications/edit without loosing a seconde !

So, I cant screenshoot from Wekan (this dont exists)
But my inspiration link (NullBoard) do it well !

The Drag n drop is so cool (much better than original Kanboard)
But it make the one click on card unusable for other thing like direct editing...
Maybe a clic+Shift/Alt for direct editing should be a solution !

Have a good coffee time
Thanks

**Comment by DimDz** [comment]
So if I understand correctly, if I would press Alt + Click on a card, it should edit the title of the card.

**Comment by bonnebulle** [comment]
Yes, exactly : editing directly

With special interaction like :
- ALT+clic 
- or just double click on it
- or a special mod
(something that not change the actual thinking-use of Wekan)

**Comment by chrisi51** [comment]
why you want to add a key press to the click instead of just clicking the title? afraid of ppl not be able anymore to drag the card successfully?
dunno if its usefull to use combinations of key or mouse strokes as ppl likely tend to forget those shortcuts.

**Comment by xet7** [comment]
WeKan is not nullboard. I think this is not useful.

**Comment by chrisi51** [comment]
well i like the idea to edit title directly on minicard without having to open it in detail mode. but i wonder why you would want to add modifier key to the mouse click event.

**Comment by xet7** [comment]
@chrisi51 

Do you mean `E` for quick edit mode like in Trello? https://trello.com/shortcuts

You can add it to WeKan if you like.

WeKan shortcut issue is at #1878

**Comment by chrisi51** [comment]
@xet7 

i have no idea about trello workflows. i think, clicking the card itself is fast enough to edit "the card attributes" if it comes to details, which are not shown on the minicard.

the user was requesting a direct edit of the title on the minicard like you can change dates already on the minicard:
![image](https://github.com/wekan/wekan/assets/2156760/b3f67e83-1ac3-48e3-ba14-e80605e0f5a3)

like this
![image](https://github.com/wekan/wekan/assets/2156760/5001c070-2b78-4b95-87fb-4b0a68b406bb)
becomes this when u click the title instead of the rest of the card
![image](https://github.com/wekan/wekan/assets/2156760/bafc8efd-603a-49b6-a188-3b7236f217a9)

(i already tested around yesterday but dit not get it working yet, so screenshot is from my dev)

**Comment by bonnebulle** [comment]
Yes a shortcut like Trello
Or 
a one click edit option 
I clic on the title part (like @chrisi51 last screenshoot)...
and it just edit it (has textarea)...
like on card creation

And, yes is WK is realy not Nullboard !
Thanks

**Comment by chrisi51** [comment]
@bonnebulle im working on it but dont got much time right now

**Comment by bonnebulle** [comment]
Hey @chrisi51 
You will work when you have time !
I realy think it can make WK better
But it's not urgent !
Thanks for all

**Comment by chrisi51** [comment]
@bonnebulle why ur closing it then? XD

**Comment by bonnebulle** [comment]
Lol, am just asking for a good improvement here.
It is not close because it is undone !
If we close it, this feature will never appear (and it's sad)
I said, "take ur time" because am not working on it ! (I dont know how to do)
So, I can close it has "non planned"
Thanks for all your working

**Comment by chrisi51** [comment]
@bonnebulle YOU are closing it again xD 

you press the wrong button or something like ... just dont close it if your not want to close it.

**Comment by bonnebulle** [comment]
;) am full noob, you gat me

**Comment by mohammadZahedian** [comment]
why this is cloesd? it is done?

**Comment by bonnebulle** [comment]
Not really.... 
For me, a quicker edition should make Wekan more effective
But if I understood well, this change will modify to much the actual UX
Thanks to reopening it

**Comment by xet7** [comment]
If any issue is important to someone, and someone comments at it, I usually open that issue.

In general, making editing quicker is a good idea. When actually sometime it will be implemented, I presume it will not modify actual default UX, it will just add some additional quicker way.
